src/Behavioral.Automation/Bindings/DropdownBinding.cs

```diff
@@ -65,12 +65,14 @@ public void CheckDropdownHeaders([NotNull] IDropdownWrapper wrapper, [NotNull] T
                 !behavior.Contains("not"), $"{wrapper.Caption} items are {items.Aggregate((x, y) => $"{x}, {y}")}");
         }
 
+        [Given("the \"(.*) was selected in (.*)")]
         [When("user selects \"(.*)\" in (.*)")]
         public void ClickOnEntry([NotNull] string entry, [NotNull] IDropdownWrapper wrapper)
         {
             wrapper.Select(entry);
         }
 
+        [Given("multiple entries were selected in (.*)")]
```

Comment:
We agreed to use phrase "user" to simulate actions.
Expected binding is:
user selected multiple entries in (.*)

Reply:
Please, update the Bindings.md

Reply:
Fixed